patches/minecraft/net/minecraft/enchantment/EnchantmentHelper.java.patch

```diff
@@ -27,3 +44,42 @@
              {
                  for (int i = enchantment.func_77325_b(); i > enchantment.func_77319_d() - 1; --i)
                  {
+@@ -429,6 +434,19 @@
+         return list;
+     }
+ 
++    /* ======================================== FORGE START =====================================*/
++
++    public static float getModifierForCreature(ItemStack stack, @javax.annotation.Nullable Entity entity)
```

Comment:
Javadoc this and describe what it does. Also, there is no need to add a Forge block like this if it's only one method.